Firestore/Source/Model/FSTMutation.h

```diff
@@ -197,6 +199,7 @@ typedef NS_ENUM(NSUInteger, FSTPreconditionExists) {
  * FSTSetMutation, but not necessarily for an FSTPatchMutation).
  */
 - (FSTMaybeDocument *_Nullable)applyTo:(FSTMaybeDocument *_Nullable)maybeDoc
+                               baseDoc:(FSTMaybeDocument *_Nullable)baseDoc
```

Comment:
Objective-C style would have us avoid this abbreviation in the label.

Reply:
I changed the label to baseDocument, kept baseDoc as the argument name though to be consistent with mayebDoc.